README.md

```diff
@@ -29,19 +17,24 @@ This is a prototype to test a few ideas on how to gather a stream of things crea
 
 - `git clone -b react https://github.com/mozilla/network-pulse.git`
 - `cd network-pulse`
+- `cp sample.env .env` (and modify values in sample.env so they match what you use for your local development env)
+    ```bash
+    HOST='the host you are using'
```

Comment:
I would recommend updating this so that it's explicit about what the values should be:

```bash
HOST=<url for the domain the pulse app runs on, without port suffix, and without trailing slash>
POST=<port number>
PULSE_API=<url for the pulse API server, without trailing slash>
```